stage_descriptions/geospatial-01-zt4.md

```diff
@@ -0,0 +1,35 @@
+In this stage, you'll add support for responding to the `GEOADD` command.
+
+### The `GEOADD` command
+The `GEOADD` command adds a location (with longitude, latitude, and name) to a key. It stores the location as a [sorted set](https://redis.io/docs/latest/develop/data-types/sorted-sets/) under the specified key. If the key doesn’t exist, a new sorted set is created under the specified name and the location is inserted to it. If the key exists, the location is inserted in the sorted set.
+
+The syntax for `GEOADD` command is
```

Comment:
Let's not use the syntax view here - show the example usage first (i.e. the "show don't tell" technique) and then if needed, explain what the arguments are further. For most simple commands users will be able to infer what the values are if we just use appropriate names